lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -162,11 +161,6 @@ def self.included(includer)
             :description => "Location of the Chef Client MSI. The default templates will prefer to download from this location. The MSI will be downloaded from chef.io if not provided.",
             :default => ''
 
-          option :install_as_service,
-            :long => "--install-as-service",
-            :description => "Install chef-client as a Windows service",
-            :default => false
```

Comment:
If we drop this feature, we need a clear message about what people should do instead and how they should get there. Maybe a blog post, and then keep this flag but have it raise a message about how this isn't supported that links to the blog post?

Should talk to @charlesjohnson about this. My gut feeling is that we're not ready to just drop this, and we need to pave a better road which will take some time.